kerberos/Dockerfile

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2017 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM centos:6
```

Comment:
is `centos:6` the latest centos6 version?

Reply:
i thought it was getting me the latest version of centos6, but now i am not so sure. i will just hard code it to the latest version of centos6

Reply:
set it explicitly to version 6.9